modules/utxo-lib/src/bitgo/transaction.ts

```diff
@@ -15,9 +15,20 @@ import { ZcashNetwork, ZcashTransaction } from './zcash/ZcashTransaction';
 export function createTransactionFromBuffer<TNumber extends number | bigint = number>(
   buf: Buffer,
   network: Network,
-  { version }: { version?: number } = {},
-  amountType: 'number' | 'bigint' = 'number'
```

Comment:
Seems to be breaking change?, Can you kindly mention in the commit message ([format](https://www.conventionalcommits.org/en/v1.0.0/#:~:text=in%20Semantic%20Versioning\).-,BREAKING,-CHANGE%3A%20a))

Reply:
it's not, old callsites will behave as before

Reply:
Got it!!, thanks a lot for making it non-breaking